plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/wrapper/LibvirtDeleteVMSnapshotCommandWrapper.java

```diff
@@ -24,6 +24,7 @@
 
 import org.apache.cloudstack.storage.to.PrimaryDataStoreTO;
 import org.apache.cloudstack.storage.to.VolumeObjectTO;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
```suggestion
import com.cloud.utils.StringUtils;
```

maybe also in another location.

Reply:
`com.cloud.utils.StringUtils` does not have method `isNotEmpty`

Reply:
`public class StringUtils extends org.apache.commons.lang3.StringUtils`

Reply:
ok, thanks @DaanHoogland 
I think these two have no difference, right ?

```
<08:32:31> weizhou:/git/cs (4.20)# git grep utils.StringUtils  |wc -l
113
<08:32:35> weizhou:/git/cs (4.20)# git grep lang3.StringUtils |wc -l
443
```

Reply:
yes, but please use it through the proxy class